### PR TITLE
[Network wathcer] add a new field trafficAnalyticsInterval for TA

### PR DESCRIPTION
**Pull request by zwswim**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/networkWatcher.json [hunk 1]
```diff
@@ -2669,7 +2669,11 @@
         },
         "workspaceResourceId": {
           "description": "Resource Id of the attached workspace ",
-         "type": "string"
+          "type": "string"
+        },
+        "trafficAnalyticsInterval": {
```
Comment (dsgouda) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/networkWatcher.json]: Do any of the examples need to be updated?

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part

File: specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/networkWatcher.json (hunk 1 above)
Comment (dsgouda) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/networkWatcher.json]: If it's an optional field this should not be an issue.

**Review by dsgouda — APPROVED** [review]
LGTM

File: specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/networkWatcher.json (hunk 1 above)
Comment (zwswim) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/networkWatcher.json]: >             If it's an optional field this should not be an issue.

Hi, Thanks for take care of the review.  Do I need to do anything like click the button "close Pull Request" after you approve the PR ?

**Pull request merged**